src/components/stepforms/widgets/InputText.vue

```diff
@@ -1,6 +1,11 @@
 <template>
   <div class="widget-input-text__container" :class="toggleClassErrorWarning">
-    <label v-if="name" :for="id">{{ name }}</label>
+    <div class="widget-input-text__header">
+      <label v-if="name" :for="id">{{ name }}</label>
+      <a v-if="docUrl" :href="docUrl" target="_blank">
```

Comment:
I think we should add [rel="noopener"](https://mathiasbynens.github.io/rel-noopener/) to links with `target="_blank"`.